### ISSUE TITLE
[BUG] A11y: Poor contrast on default color scheme

### ISSUE THREAD

**Issue by mbmjertan** [infinum/eightshift-frontend-libs#490]
## Describe the bug

We have pretty poor contrast with the default color scheme on components such as buttons and lists:
<img width="260" alt="Screenshot 2021-11-10 at 16 57 53" src="https://user-images.githubusercontent.com/1742806/141147435-ab0b12b3-38a4-46a8-8275-a5bc1315ca93.png">

According to IBM's Accessibility Checker, we have a contrast ratio of 3.55 to 1, which is a violation of WCAG 2.1 AA requirements:

> Text contrast of 3.55 with its background is less than the WCAG AA minimum requirements for text of size 16px and weight of 700

<img width="841" alt="Screenshot 2021-11-10 at 17 00 10" src="https://user-images.githubusercontent.com/1742806/141147880-6e1982f2-a19c-402b-94c6-ff0cca0d27b6.png">
